pages/common/hydra.md

```diff
@@ -12,7 +12,7 @@
 
 `hydra -l {{username}} -P {{path/to/wordlist.txt}} {{host_ip}} {{ssh}}`
 
-- Guess HTTPS webform credentials using a list of usernames and a list of passwords:
+- Guess HTTPS webform credentials using a list of usernames and a list of passwords (`{{https_post_request}}` can be like `username=^USER^&password=^PASS^`):
```

Comment:
Did you try this in a client with `--render`? I think this token syntax is not valid. It doesn't work in my tldr client made in Rust.

Reply:
I use a different client: *Haskell tldr client*. It works perfectly fine with my client. But yes, with the *Rust tealdeer client*, I see formatting issues. Given that the markdown itself is correct, don't you think this is a bug in the rust client itself? 

Let me know what you think and I will try to find an alternative solution.

(I have attached a screenshot comparing the Haskell and Rust clients) 
![hydra](https://user-images.githubusercontent.com/1625934/169700101-2312acdf-c6a9-4301-ad24-4e75c6c30bd9.png)

Reply:
I just tried with the official [tldr client built with python](https://github.com/tldr-pages/tldr-python-client) and I do see the same result as the rust client. Confused now.

Reply:
@dbrgn could you please give us your opinion? is it a bug?

Reply:
In my opinion it is not a bug, because the [original specification](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md) does not indicate that descriptions can contain tokens. In the case of tealdeer, the client does not implement full CommonMark support, because that would be *way* more complex (and could never be fully supported because any HTML is also valid CommonMark, and you cannot render HTML in the console).

The [current version of the client spec](https://github.com/tldr-pages/tldr/blob/main/CLIENT-SPECIFICATION.md#page-structure) says:

> Although this specification is about the interface that clients must provide, it is also worth noting that pages are written in standard [CommonMark](https://commonmark.org/), which the exception of the non-standard `{{` and `}}` syntax, which surrounds values in an example that users may edit. Clients MUST NOT break if the page format is changed within the _CommonMark_ specification.

Tealdeer does not break when CommonMark is used in the description, but it is simply not rendered. This is in line with the spec.

I would suggest to avoid any type of formatting in descriptions.

Reply:
@dbrgn but there are some pages with highlights in the example description (`common/csvcut.md`, `common/strings.md` and `common/supervisorctl.md`) and the italic is not rendered in the terminal.